packages/paste-website/src/pages/index.tsx

```diff
@@ -46,7 +47,7 @@ const IndexPage: React.FC<{}> = (): React.ReactElement => {
         </Heading>
         <P variant="lead">
           Paste is a design system used to build accessible, consistent, and high quality customer experiences at
-          Twilio. Paste is open source and contributions are welcome.
+          Twilio. Paste is open source and contributions are welcome. <BrowserSupport />
```

Comment:
Not sure if this is where this should live on the home page. Any thoughts on moving to the footer?